src/lektorium/repo/local/storage.py

```diff
@@ -555,7 +563,7 @@ def create_site(self, lektor, name, owner, site_id):
 
         return site_workdir, options
 
-    def create_site_repo(self, site_id):
+    async def create_site_repo(self, site_id):
```

Comment:
init_project should also be async

Reply:
I intended to work on Gitlab and AWS in a related [issue](https://spherical.atlassian.net/browse/IARCRP-88)